src/ebay-button/button.tsx

```diff
@@ -66,8 +66,7 @@ const EbayButton:FC<Props> = ({
     const sizeStyles: { [key in Size]: string } = {
         large: `${classPrefix}--large`,
         small: `${classPrefix}--small`,
-        regular: '',
-        default: ''
+        regular: ''
```

Comment:
Is there a reason to keep `regular` even though it has no styles? Maybe `Size` should just be `'small' | 'large' | undefined`

Reply:
That is true, it is not necessary since skin doesn't use. @agliga should we remove on Marko for v15 also?

Reply:
Actually marko, doesnt' have it anymore. Let me remove it